src/catalog/src/durable/transaction.rs

```diff
@@ -554,6 +562,56 @@ impl<'a> Transaction<'a> {
         Ok(())
     }
 
+    pub fn insert_builtin_network_policy(
```

Comment:
Is this used? If so, should it be `insert_system_network_policy(...)`?

Reply:
not used yet, I'll remove. When it gets re-introduced it won't be to create a system policy, it'll create a pre-defined user policy.